store/tikv/2pc.go

```diff
@@ -527,7 +548,7 @@ func (c *twoPhaseCommitter) doActionOnBatches(bo *Backoffer, action twoPhaseComm
 		return nil
 	}
 
-	if len(batches) == 1 {
+	if len(batches) == 1 || action.inRetry() {
```

Comment:
Why only handle the first batch?
Other batches are left uncommitted?

Reply:
orz, I write a bug